slack/plugin.py

```diff
@@ -264,6 +275,17 @@ def slack_conversations_history(
             .json()
         )
     if not data.get("ok", False):
+        if data.get("error") == "channel_not_found":
+            # Avoid making the error message absurdly long.
+            available_channels = (
+                f"available channels: {sorted(list(conversation_ids.keys()))}; "
+                if len(conversation_ids) < 256
+                else ""
```

Comment:
what about truncating with "..." instead of dropping completely?  Giving some might help the user / Lutra figure out they've authorized a different workspace, for example.

Reply:
but i was being lazy